examples/docker/.dockerignore

```diff
@@ -0,0 +1,4 @@
+**/build
+**/.cache
+**/node_modules
+**/npm-debug.log
```

Comment:
You'll likely want to remove `.git` folder as well

Reply:
i will add `.git` too